### PR TITLE
update the README

### PR DESCRIPTION
**Pull request by nimakaviani**
[#162409859]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
 
-Eirini is a Kubernetes backend for Cloud Foundry. It deploys CF apps to a kube backend, using OCI images and Kube deployments.
+*Eirini* is a Kubernetes backend for Cloud Foundry.
+It deploys CF apps to a kube backend, using OCI images and Kube deployments.
 
-_But there's more!_
+Eirini exports staged CF images as docker images.
+So you can schedule them however you'd like.
+*And separately* it gives you the nice integrated `cf push` flow,
+with CF Apps mapped directly to kube deployment objects (pods).
```
Comment (suhlig) [inline README.md]: Currently, we actually map each CF app to a `StatefulSet`.

File: README.md [hunk 2]
```diff
@@ -1,44 +1,60 @@
...
 
-# Eirini - what this?
+# What is Eirini?
 
-Eirini is a Kubernetes backend for Cloud Foundry. It deploys CF apps to a kube backend, using OCI images and Kube deployments.
+*Eirini* is a Kubernetes backend for Cloud Foundry.
+It deploys CF apps to a kube backend, using OCI images and Kube deployments.
 
-_But there's more!_
+Eirini exports staged CF images as docker images.
```
Comment (suhlig) [inline README.md]: This is no longer true; the bits-service handles that.

File: README.md [hunk 3]
```diff
@@ -1,44 +1,60 @@
...
+with CF Apps mapped directly to kube deployment objects (pods).
+In other words it decouples buildpack staging and stateless-multitenant-app running.
 
-Eirini exports staged CF images as docker images. So you can schedule them however you'd like. *And separately* it gives you the nice integrated `cf push` flow, with CF Apps mapped directly to kube Deployment objects. In other words it decouples buildpack staging and stateless-multitenant-app running.
+Since scheduling is increasingly commoditized,
+Eirini provides an "Orchestrator Platform Interface (OPI)" layer,
+that abstracts away orchestration from Cloud Foundry's control plane.
+This means Eirini is not solely a Kube backend at all,
+but that it is a generic backend for any scheduler!
+This means it can schedule to diego, kube, swarm and other orchestration platforms,
```
Comment (suhlig) [inline README.md]: We'd prefer `could` over `can` because in reality, there only is Kubernetes.

File: README.md [hunk 4]
```diff
@@ -1,44 +1,60 @@
...
+as long as there is an implementation of the OPI layer for the target platform.
 
-_But there's more!_
+To offer a generic orchestrator interface,
+Eirini uses the diego abstractions of LRPs and Tasks to capture Cloud Foundry's notion of long running processes and one-off tasks.
 
-Eirini uses a little abstraction library, "OPI", which means it's not actually a Kube backend at all: it's a generic backend for any scheduler! This means it can schedule to diego/kube/swarm and whatever else is cool next year.
+An experimental BOSH release for Eirini is available at [cloudfoundry-incubator/eirini-release](https://github.com/cloudfoundry-incubator/eirini-release)
```
Comment (suhlig) [inline README.md]: The develop branch already updated this to

> Deployment instructions are available at https://github.com/cloudfoundry-incubator/eirini-release

File: README.md [hunk 5]
```diff
@@ -1,44 +1,60 @@
...
-# What components?
+# Eirini Components
 
 Eirini has the following components, the first two are available as subcommands of the `eirini` binary:
  
  - `Bifrost` converts and transfers cloud controller app specific requests to OPI specific objects and runs them in Kubernetes. It relies on the `Registry` to serve OCI images for droplets, and `OPI` to abstract the communication with Kube. 
- - `OPI` or the "orchestrator provider interface" provides a declarative abstraction over multiple schedulers inspired by Diego's LRP/Task model and Bosh's CPI concept.
+ - `OPI` or the "Orchestrator Provider Interface" provides a declarative abstraction over multiple schedulers inspired by Diego's LRP/Task model and Bosh's CPI concept.
  - `St8ger` implements Staging by running Kubernetes/OPI one-off tasks
+ - `Registry` provides a private docker registry that serves application OCI images created after staging the application droplet. While this used to be an Eirini component in the earlier versions of Eirini, the registry service is now offered as part of deploying [cloudfoundry-incubator/bits-service](https://github.com/cloudfoundry-incubator/bits-service) in the Eirini helm chart.
```
Comment (suhlig) [inline README.md]: It's gone.

File: README.md [hunk 6]
```diff
@@ -1,44 +1,60 @@
...
+Specifically, these are Long Running Processes (LRPs) and Tasks.
+Actually, LRPs and Tasks are most of what you need to build a PaaS,
+and they're cross-cutting concepts that map nicely to all current orchestrators
+(for example to LRPs/Tasks directly in Diego,
+to Deployments/Jobs in Kube,
+and to Services/Containers in Swarm).
+
+One of the great things about BOSH is the CPI abstraction that lets it work on any IaaS.
+Cloud Foundry however, has been tightly coupled to one specific Orchestrator (Diego).
+This was fine for fast iteration, but now that orchestration is increasingly commoditized
```
Comment (suhlig) [inline README.md]: The "commoditized" statement is somewhat repeated from above, but we actually think that the whole sentence could be removed.

File: README.md (hunk 1 above)
Comment (nimakaviani) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (nimakaviani) [reply README.md]: sure I can remove it. I asked this question on slack on whether it makes sense to have the registry definition back with a slight twist on how it is replaced by bit services: https://cloudfoundry.slack.com/archives/C8RU3BZ26/p1546897685022500?thread_ts=1546897659.022400&cid=C8RU3BZ26

but if it makes more sense to remove it, I can nuke it.

File: README.md (hunk 3 above)
Comment (nimakaviani) [reply README.md]: see above

File: README.md (hunk 5 above)
Comment (nimakaviani) [reply README.md]: see above

File: README.md (hunk 2 above)
Comment (suhlig) [reply README.md]: Thanks and sorry, I did not tie the two things together. Yes, mentioning the role of the bits-service and pointing to it makes perfect sense.

File: README.md [hunk 7]
```diff
@@ -1,40 +1,54 @@
...
 
 Eirini has the following components, the first two are available as subcommands of the `eirini` binary:
  
  - `Bifrost` converts and transfers cloud controller app specific requests to OPI specific objects and runs them in Kubernetes. It relies on the [`bits-service`](https://github.com/cloudfoundry-incubator/bits-service) to serve OCI images for droplets, and `OPI` to abstract the communication with Kube.
- - `OPI` or the "orchestrator provider interface" provides a declarative abstraction over multiple schedulers inspired by Diego's LRP/Task model and Bosh's CPI concept.
+ - `OPI` or the "Orchestrator Provider Interface" provides a declarative abstraction over multiple schedulers inspired by Diego's LRP/Task model and Bosh's CPI concept.
  - `Stager` implements staging by running Kubernetes/OPI one-off tasks
  
-# Tell me more 'bout OPI
+# Orchestrator Platform Interface (OPI)
```
Comment (suhlig) [inline README.md]: Just recognised this - I believe we call it Orchestrator ~Platform~**Provider** Interface.

**Pull request merged**